app/containers/App/tests/index.test.js

```diff
@@ -1,11 +1,15 @@
+import App from '../index';
+import Footer from 'components/Footer';
+
 import expect from 'expect';
 import { shallow } from 'enzyme';
 import React from 'react';
 
-import App from '../index';
-import Footer from 'components/Footer';
-
 describe('<App />', () => {
+  /* eslint no-underscore-dangle: ["error", {"allow": ["__Rewire__", "__ResetDependency__"]}] */
```

Comment:
Let's move this to the `package.json` and allow `__Rewire__` and `__ResetDependency__` globally – I have some other ideas where we could use rewire! (e.g. reducer initial state)